### PR TITLE
Add AKS onboarding form

### PR DESCRIPTION
**Pull request by malcolmbaig**
### Context

We're migrating AYTQ/CTR to AKS. 

### Changes proposed in this pull request
Kick off the process by completing this onboarding form ([source](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-form-template.md)) to share with [teacher_services_infra](https://ukgovernmentdfe.slack.com/archives/C011EM7HU85).

### Link to Trello card
https://trello.com/c/Ng21PrA6/130-start-aytq-aks-migration-by-completing-onboarding-form-and-generating-initial-code

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: docs/aks/onboarding-form.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+two different domains, and are currently deployed to (non-AKS) Azure.
+
+- Service name? Access your teaching qualifications / Check a teacher's record (two services, single codebase)
+- Domains ?
+  - access-your-teaching-qualifications.education.gov.uk (already exists)
+  - check-a-teachers-record.education.gov.uk (already exists)
+  - access-your-teaching-qualifications.teacherservices.cloud
+  - check-a-teachers-record.teacherservices.cloud
+  - test._ and preprod._ variants of the above.
+- Service short name? aytqctr
```
Comment (malcolmbaig) [inline docs/aks/onboarding-form.md]: The existing service short name is just aytq — using aytqctr here so it's a little more obvious that this app consists of two services.

File: docs/aks/onboarding-form.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Repository? https://github.com/DFE-Digital/access-your-teaching-qualifications/
+- Environments? e.g. develoment, staging, production
+  - test
+  - preprod
+  - production
```
Comment (malcolmbaig) [inline docs/aks/onboarding-form.md:18]: @felixclack I've opted not to have a dev environment here as it's always seemed a little redundant. Children's barred list took the same approach when it was set up on AKS.

Comment (felixclack) [reply docs/aks/onboarding-form.md:18]: Makes sense to me. I never used the deployed dev environment.

File: docs/aks/onboarding-form.md (hunk 1 above)
Comment (saliceti) [reply docs/aks/onboarding-form.md]: I think it's limited to 6 characters. Maybe atqctr? or tqtr? or tqr?
Or keep aytq as it's the name of the Azure Product tag and there can be only 1.

File: docs/aks/onboarding-form.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Review apps? Yes or No
+  - Yes
+- Cost centre/Activity code ? e.g. 10167/101304
+  - Unknown
+- List of developers who will need access to Azure? Do they use a DfE laptop or BYOD? e.g
+  - Felix Clack - BYOD
+  - James Gunn - BYOD
+  - Richard Pattinson - DfE laptop
```
Comment (saliceti) [inline docs/aks/onboarding-form.md:26]: @malcolmbaig you are owner of the `s189 TRA delivery team` group. You can add Richard as member

File: docs/aks/onboarding-form.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  - Yes
+- Slack channels? e.g
+  - Product: #tra_digital
+  - Tech: #tra_developers
+- Timeline? When is each environment needed? What are the planned project phases?
+  - No specific timeline, will be working on Review apps first with assistance from infra team, followed by test/preprod/production.
+- Github repos / Docker registry type: Open (public) by default, or indicate any requirement to make them private
+  - Open
+- Namespaces e.g. bat-qa (test cluster), bat-staging (test cluster), bat-production (production cluster)
+  - n/a
```
Comment (saliceti) [inline docs/aks/onboarding-form.md]: You will most certainly use the tra namespaces. Today there are: tra-development, tra-test, tra-production. You can decide how to map the environments to the namespaces.

File: docs/aks/onboarding-form.md (hunk 1 above)
Comment (malcolmbaig) [reply docs/aks/onboarding-form.md]: Let's keep aytq 👍🏼

**Pull request closed without merging**